### DISCUSSION TITLE
Unable to change the size in 0.4.0

### DISCUSSION THREAD

**Discussion by JemmyWang94** [rustdesk/rustdesk#9044] [Q&A]
In version 0.3.9 of pub, I was able to drag the mouse at the edge to change the size of the application, but after updating to 0.4.0, I was unable to change the size of the application by dragging the edge.

**Comment by fufesou** [comment]
https://github.com/leanflutter/window_manager/issues/483

@JemmyWang94 Please close this discussion. I guess you may have accidentally posted to the wrong place.
